lte/gateway/c/core/oai/test/spgw_task/mock_spgw_op.cpp

```diff
@@ -0,0 +1,49 @@
+#include <fstream>
```

Comment:
For all new files added, please add Magma copyright header (you can pick it up from already added files)